.evergreen/config.yml

```
@@ -170,6 +170,22 @@ functions:
           # initialize submodules
           git submodule init
           git submodule update
+    - command: shell.exec
+      params:
+        working_dir: src/go.mongodb.org/mongo-driver
+        shell: "bash"
+        script: |
+          . ${DRIVERS_TOOLS}/.evergreen/utils.sh
```

```suggestion
          . ${DRIVERS_TOOLS}/.evergreen/venv-utils.sh
```

`venv-utils.sh` is the hygienic alternative.